lib/karafka/processing/worker.rb

```diff
@@ -53,28 +53,30 @@ def process
         instrument_details = { caller: self, job: job, jobs_queue: @jobs_queue }
 
         if job
-          Karafka.monitor.instrument('worker.process', instrument_details)
+          job.wrap do
+            Karafka.monitor.instrument('worker.process', instrument_details)
```

Comment:
Can we also import `monitor` here and refer to it only with `monitor` instead of `Karafka.monitor`?